docs/specification/current.md

```diff
@@ -1095,6 +1095,76 @@ As each hook progresses through a process of being defined, tested, implemented,
 
     hookMaturity | 0 - Draft
 
+## Coordination 
+The primary motivation for configuration options is to support the functionality of coordinating between `order-select` and `order-sign` to prevent alert duplication. Although it might seem reasonable to have the service default to this kind of coordination with no input from the EHR client, we think that there might be potential scenarios where this form of duplication might be desirable. For example, a teaching hospital might have resident physicians go through the order selection process and then have an attending physician sign off on the order. In this situation, having the same CDS Hooks cards returned at `order-select` and `order-sign` would not be perceived as a duplication to the attending. There might be other scenarios where repeating cards across the hooks makes sense. Thus, it seems best to have the EHR client some active role in the decision of whether to filter out alert responses. Using configuration options sent from the EHR client to the service provides a way for the client to have a say in how apparent duplication is handled.
+
+This implementation guide discusses a mechanism for providing the CDS service configuration from CDS Hooks requests. The use of this mechanism is required for the functionality of coordinating between order-select and order-sign. There is no requirement to use additional configuration options at this time. 
```

Comment:
> This implementation guide...

Because the base spec is not an implementation guide, any proposals should be more generically applicable across a variety of hooks and usage of those mechanisms would be defined in an implementation guide, such as PDDI.